CHANGELOG.md

```diff
@@ -5,17 +5,23 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#38, #44)
 
 New features:
 
 Bugfixes:
 
 Other improvements:
+- Fixed `Foldable1 (NonEmpty f)` instance, as `Foldable1` now has `foldr1` and `foldl1` as members (#39)
```

Comment:
Technically this was bundled into #38, but the standalone PR was #39 so in this case I opted to keep that reference.